src/rt/sched/schedulerthread.h

```diff
@@ -187,6 +201,14 @@ namespace verona::rt
             << "Should steal for fairness!" << Systematic::endl;
           should_steal_for_fairness = true;
         }
+
+        // Poll for IO periodically
```

Comment:
Should the polling occur after the executing the messages.  The messages may be removing the pollers.  Moreover, is there a semantic guarantee that a poller will run for one last time after it has been removed, so it will see everything up to the point where it is removed.